packages/easy_logging/easy_logging.0.1/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+synopsis: "Module to easily log messages"
+description: """
+     Logging infrastructure inspired by the Python logging module.
+The aim of this module is to provide a quick and easy to use logging
+infrastructure.
+
+It has the following features :
+   * one line logger creation
+   * log messages are either [string] or [string lazy_t]
+   * log level adaptable at runtime from anywhere in the program
+   * handlers associated to each logger will format and treat the message independantly
+"""
+maintainer: "mathiasmillet@gmail.com"
+authors: "Mathias Millet"
+license: "GPL"
+homepage: "https://sapristi.github.io/easy_logging/easy_logging/"
+bug-reports: "https://github.com/sapristi/easy_logging/issues"
+dev-repo: "git+https://github.com/sapristi/easy_logging.git"
+depends: [ "ocaml" "dune" "ppx_deriving"]
```

Comment:
dune needs to be tagged as a `{build}` dependency. Moreover, one advice I could give is that it would be more readable if the dependencies were indented.

Reply:
I also see that your `dune` file in the source repository uses `batteries` which is not listed in the dependencies

Reply:
Thanks, I'll fix that.
I remove batteries from the code but forgot about dune files.